joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/ast/ConditionalTests.scala

```diff
@@ -0,0 +1,84 @@
+package io.joern.csharpsrc2cpg.querying.ast
+
+import io.joern.csharpsrc2cpg.testfixtures.CSharpCode2CpgFixture
+import io.shiftleft.codepropertygraph.generated.ControlStructureTypes
+import io.shiftleft.codepropertygraph.generated.nodes.{ControlStructure, Identifier, Literal}
+import io.shiftleft.semanticcpg.language.*
+class ConditionalTests extends CSharpCode2CpgFixture {
+
+  "AST Creation for conditionals" should {
+    "be correct for if statements" in {
+      val cpg = code(basicBoilerplate("""
+          |int a = 1;
+          |if (a == 1) {
+          | a++;
+          |}
+          |""".stripMargin))
+
+      cpg.method("Main").controlStructure.size shouldBe 1
+
+      inside(cpg.method("Main").controlStructure.l) { case List(ifNode: ControlStructure) =>
```

Comment:
You can also use the style `case ifNode :: Nil => ...`. Since you call `cpg.method("Main").controlStructure`, these are expected to all be control structures, so no need to check.

Also, could you add a default case which fails the test, please? Something like `case _ => fail("No control structure found!")`. Similarly with the tests below.